rialto_airflow/dags/harvest.py

```diff
@@ -220,50 +221,22 @@ def upload_data_quality_files(snapshot):
                 str(file_path), google_folder_id
             )
 
-    # link up dag tasks using their outputs as dependencies
-
-    snapshot = setup()
-
-    snapshot = load_authors(snapshot)
-
-    sul_pub_jsonl = sul_pub_harvest(snapshot)
+    @task_group()
+    def upload(snapshot):
+        upload_open_access_files(snapshot)
+        upload_data_quality_files(snapshot)
 
-    dimensions_jsonl = dimensions_harvest(snapshot)
+    # link up dag tasks and task groups
 
```

Comment:
All of this gets greatly simplified since we are linking the tasks together with `>>` below instead of using the task outputs to express workflow dependencies.